config/nrf52_pins.h

```diff
@@ -68,34 +66,30 @@
     #define SX1276_RXTX_PIN 30
 
     // RX/TX pins for the GPS UART
-    #define GPS_RX_PIN 17
-    #define GPS_TX_PIN 18
+    #define GPS_RX_PIN 29
```

Comment:
I haven't touched the PIN definitions in the NRF52_DK section (I assume that they are correct). It looks like the layout for EE03 has switched these pins. EE03 communicates fine with the GPS using the definitions in the EE02 block.